bsp4s/src/org/jetbrains/bsp/util/CustomCodec.scala

```diff
@@ -1,4 +1,4 @@
-package ch.epfl.scala.utils
+package org.jetbrains.bsp.util
```

Comment:
Why did you change the package of this file? This is a breakage of the binary compatibility. Which means that in theory we should bump the major version of the library.

Reply:
accidentally...